scene/main/node.cpp

```diff
@@ -2673,12 +2673,24 @@ void Node::clear_internal_tree_resource_paths() {
 	}
 }
 
-String Node::get_configuration_warning() const {
+Array Node::get_configuration_warning() const {
```

Comment:
This should likely be `get_configuration_warnings` or `get_configuration_warnings_list`.